src/components/EditCutPacking.tsx

```diff
@@ -141,8 +141,7 @@ class EditCutPacking extends React.Component<Props, State> {
     } = this.state;
 
     if (this.state.redirect) {
-      redirect(`/cutPackings`);
-      return null;
+      return <Navigate replace={true} to={`/cutPackings`}/>;
```

Comment:
No variable in the 'to', so do you need the template literal, could it just be a string as in other components?